recipes/vdom/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "vdom" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d43fb8e92e64e1cabbdb93b4f213b95145db0e843371239b3a8bf91dbdf65159"
+
+build:
+  number: 0
```

This looks like it could be noarch?
```suggestion
  number: 0
  noarch: python
```

---

I'm not sure, it's just a vanilla Python library so I generated the recipe using conda skeleton.